### DISCUSSION TITLE
Submission Error

### DISCUSSION THREAD

**Discussion by weijinbao1998** [facebookresearch/fastMRI#270] [Q&A]
why could't I upload my reconstructions result?I had always compressed my file into zip format and used dropbox to save,but it is not working right now.The error message is :BadZipFile: File is not a zip file. I have checked the file format and it is zip format.
I am doing the same procedure used to do,but it is not working right now. Can you let me know why it is happening?thank you so much,I am looking forward to your reply.

**Comment by mmuckley** [comment]
Hello @1025754824, this was because you had a trailing space in your URL. I think it should work now.
